avalon/tools/cbloader/app.py

```diff
@@ -89,12 +101,15 @@ def __init__(self, parent=None):
             }
         }
 
+        filters.checkbox_list.itemChanged.connect(self.on_filter_toggled)
```

Comment:
Should it maybe just be the FilterWidget emitting a filter changed signal? That could simplify things here?

That way the logic of the filter widget is contained - it doesn't need to know about anything else, it will just tell  you which families are selected. Maybe it's not a filter widget at all, maybe it should just be a `FamiliesWidget` that emits a `selection_changed` signal?

Reply:
Fixed in latest commit